src/components/CheckoutOrderDetails/CheckoutOrderDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactElement } from 'react';
+import { useLocalOrder } from 'context/OrderContext';
+import cn from 'classnames';
+import { GBPFormat } from 'utilities/GBPFormat';
+
+export const CheckoutOrderDetails = (): ReactElement => {
+  const { localOrder } = useLocalOrder();
+
+  const iconClass = 'border rounded bg-gray-200 p-2';
+
+  const tableElement = 'whitespace-nowrap px-6 py-4 text-sm font-medium text-gray-900';
+
+  return (
+    <div className="font-sans text-[16px]">
```

Comment:
Could this be modularised to make the container/form reusable then use a details file to populate the container?

Reply:
Yeah can do! Do you want to add it yourself so you can get started familiarising yourself with the front-end? @DWatson1993

Reply:
I've taken out the use of the context here (an renamed which might be a bit confusing) so one can use any set of static Line-Items so it can be used again in the admin page etc.